rasa_sdk/forms.py

```diff
@@ -21,6 +23,8 @@
 
 LOOP_INTERRUPTED_KEY = "is_interrupted"
 
+VALIDATE_GLOBAL_SLOT_MAPPINGS_NAME = "validate_global_slot_mappings"
```

Comment:
In this case, should I update the action name searched by default action in rasa-oss to this one?